### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by RaphDasilva**
## I was able to do the following.

- [ ] Created the owners and the species table.
- [ ] Deleted the species column in the animals table.
- [ ] Added owners_id and species_id columns to the animals table and make them foreign keys linking the owners and species tables to it.
- [ ] Input data into the owners and species tables.
- [ ] Ran some complex queries using JOIN. The queries are thus listed below:

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,4 +11,33 @@ CREATE TABLE animals (
...
+ALTER TABLE animals 
+ADD COLUMN species_id INT;
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id)
+REFERENCES species(id);
+
+ALTER TABLE animals 
+ADD COLUMN owner_id INT;
```
Comment (cynthiainga) [inline schema.sql:43]: - [ ] Good job so far but the column `owner_id` should have a foreign key referencing the `owners` table, please kindly fix that 🙃 🙃

File: data.sql [hunk 2]
```diff
@@ -31,4 +31,46 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
...
 
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
-VALUES ('Ditto', '2022-05-14', 22, 'true', 4);
\ No newline at end of file
+VALUES ('Ditto', '2022-05-14', 22, 'true', 4);
+
+-- Multiple tables
+INSERT INTO owners (full_name, age) 
+VALUES ('Sam Smith',34 ),
+       ('Jennifer Orwel',19 ),
```
Comment (cynthiainga) [inline data.sql:39]: - [ ] Please kindly correct the name here, is `Jennifer Orwel` instead of `Jennifer Orwel` 😁 😁

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @RaphDasilva, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Descriptive PR ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Good commit history ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @RaphDasilva ,

Great job so far! 

May I highlight the following: 
- [x] The project is set correctly✔️ 
- [x] Well done implementing the previous reviewer's requirements✔️ 

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**